### ISSUE TITLE
Buttons do not have an accessible name

### ISSUE THREAD

**Issue by gminova** [frannyfra/frannyfra#2]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.  

Failing Elements:   
< button class="pause-button"></button >  
< button class="play-button"></button >
